# This represents a key frame

Header header
uint32[] tracked_ids
Feature[] features

